src/reducer.js

```diff
@@ -173,6 +183,30 @@ export function resourceReducer(state, action) {
       );
     }
 
+    case UPDATE_COLLECTION_FINISHED: {
+      const {
+        id, resource: resourceType, data, opts,
+      } = action;
+      const collectionIdHash = getCollectionIdHash(id);
+      const queryHash = getQueryHash(opts);
+      const idKey = getIdKey(resourceType);
+      const resourceMap = Array.isArray(data)
+        ? data.reduce((map, resource) => {
+          const resourceIdHash = getResourceIdHash(resource[idKey]);
+          return Object.assign(map, { [resourceIdHash]: resource });
+        }, {}) : {};
+      const associatedIds = Object.keys(resourceMap);
+      return state.withMutations((resourceState) => resourceState
+        .deleteIn(['updating', collectionIdHash, queryHash])
+        .update('updating', (updating) => (updating.get(collectionIdHash, iMap()).isEmpty() ? updating.delete(collectionIdHash) : updating))
+        .mergeIn(['items'], fromJS(resourceMap))
+        .setIn(
```

Comment:
Similar to `UPDATE_FINISHED` I don't think we should change the list of `associatedIds` in case we are given more or less items from the response.

Reply:
What does the list of `associatedIds` mean? On the surface level thought, as long as the update changes the values that are associated to hashed ids, I think we could remove this.

Reply:
It's what maps the items when requesting a collection when multiple collections are queried - its the list of items that belong to that exact collection.